product-service/.env

```diff
@@ -0,0 +1,5 @@
+PG_HOST=rs-product-db.cdaasqcdg9tj.eu-west-1.rds.amazonaws.com
```

Comment:
Very dangerous to commit your db connection data.
And in task 4.2: Do not commit your environment variables in serverless.yml to github!